Description: the name NrpeBooster is used by shinken
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/debian/etc/modules/booster_nrpe.cfg
+++ b/debian/etc/modules/booster_nrpe.cfg
@@ -5,6 +5,6 @@
 # module. It will bypass the launch of check_nrpe binary, and will do the
 # connection itself. You should not use check_nrpe binary with Shinken.
 define module {
-    module_name     booster-nrpe
+    module_name     NrpeBooster
     module_type     nrpe_poller
 }
